media/css/firefox/privacy/products.scss

```diff
@@ -0,0 +1,298 @@
+// Picto Cards
+
+.privacy-products-etp {
+    background-color: $color-off-black;
+
+    &> .mzp-l-content {
+        padding-top: 0;
+    }
+
+    // Custom Overrides pending https://github.com/mozilla/protocol/issues/382
+    .mzp-c-card-picto {
+        @include bidi(((text-align, left, right),));
+
+        .mzp-c-card-picto-content {
+            padding-top: 80px + $spacing-lg;
+
+            &:before {
+                @include bidi(((left, 0, right, 0),));
+                background-color: transparent;
+                background-repeat: no-repeat;
+                margin: 0;
+            }
+        }
+
+        &.invisible .mzp-c-card-picto-content:before {
+            @include background-size(103px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-invisible.svg');
+            height: 80px;
+            width: 103px;
+        }
+
+        &.control .mzp-c-card-picto-content:before {
+            @include background-size(62px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-control.svg');
+            height: 80px;
+            width: 62px;
+        }
+
+        &.protection .mzp-c-card-picto-content:before {
+            @include background-size(94px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-protection.svg');
+            height: 80px;
+            width: 94px;
+        }
+
+        @media #{$mq-md} {
+            .mzp-c-card-picto-content:before {
+                @include bidi(((left, $spacing-lg, right, 0),));
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Tracker Counter Call Out
+
+.privacy-products-tracker-counter.mzp-t-dark {
+    background-color: $color-violet-80;
+
+    .mzp-c-call-out-title {
+        margin: 0 auto $spacing-lg;
+        max-width: 24em;
+
+        @media #{$mq-md} {
+            @include font-size(32px);
+        }
+
+        @media #{$mq-lg} {
+            @include font-size(40px);
+        }
+    }
+
+    .mzp-c-cta-link {
+        color: $color-blue-30;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Product List Feature Cards
+
+.privacy-products-list {
+    margin: $layout-md auto;
+
+    .mzp-c-card-feature {
+        margin-bottom: $layout-xl;
+
+        &.has-logo {
+            .mzp-c-card-feature-title {
+                @include bidi(((background-position, top left, top right),));
+                @include image-replaced;
+                background-repeat: no-repeat;
+                margin-bottom: $spacing-lg;
+                min-height: 40px;
+            }
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    .mzp-c-card-feature-desc {
+        margin-bottom: $spacing-xl;
+    }
+
+    .mzp-c-card-feature-media img {
+        max-width: 320px;
```

Comment:
I do this (restrict feature card image width on small screen sizes) a lot. Do you think it's worth adding it to the base Protocol styles?

Reply:
I can see quite a few bedrock pages where this isn't the case, or it at least varies in size.